internal/config/config.go

```diff
@@ -16,11 +17,11 @@ const (
 )
 
 type Config struct {
-	Database   *dbConfig         `json:"database,omitempty"`
-	Service    *svcConfig        `json:"service,omitempty"`
-	KV         *kvConfig         `json:"kv,omitempty"`
-	Auth       *authConfig       `json:"auth,omitempty"`
-	Prometheus *prometheusConfig `json:"prometheus,omitempty"`
+	Database   *dbConfig          `json:"database,omitempty"`
+	Service    *svcConfig         `json:"service,omitempty"`
+	KV         *kvconfig.KvConfig `json:"kv,omitempty"`
```

Comment:
maybe we could make the package more general so could do this.
```suggestion
	KV         *kv.Config `json:"kv,omitempty"`
```